src/AppStartup.tsx

```diff
@@ -35,7 +40,7 @@ export function AppStartup() {
 
         if (isMounted()) {
           setInitialized(true);
-          const projectDir = getCachedSetting('project.currentDir');
+          const projectDir = getCachedSetting('project.current_directory');
```

Comment:
Python & TS share types for `SettingsSchema` now so there are a few changes like this. If we really want to camelcase everything then we should do it with some kind of middleware. But I'd rather have fewer types.